pages/linux/sfill.md

```diff
@@ -0,0 +1,24 @@
+# sfill
+
+> Secure overwrite free space and inodes of the partition where the specified directory resides.
```

Comment:
Perhaps this will read better?
```suggestion
> Securely overwrite the free space and inodes of the partition where the specified directory resides.
```

Reply:
maybe without _*the*_ in _the free space_

Reply:
@Waples I'm not sure about that—it helps `of the partition` later on.